ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java

```diff
@@ -665,8 +665,13 @@ else if (revoltName.equals("no")) {
                             output[0] = output[0].substring(3,output[0].length()-1);
 
                         }
+                        
+                        String adjName = tag.replace("_NAME","");
+                        String feuName = tag.replace("_FEUDATORY_NAME","_FEUDATORY_ADJECTIVE"); //for certain mission tags which have different formatting
```

Comment:
Wouldn't ```feuAdj``` be closer to what you're doing? better yet, use the full words: ```feudatoryAdjective```

Reply:
I could use `feudatoryAdjective` instead, but `feuAdj` was quicker to type, lol, and my comment provides explanation on it's usage

Reply:
It's easy to type, but is harder to understand when reviewing or maintaining. Abbreviations are a bad idea. See Effective Java Item 68 (Google it if you don't have a copy).